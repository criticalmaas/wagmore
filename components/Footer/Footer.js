import React from "react"
import { Grid, Row, Col } from "react-bootstrap"

const Footer = () => (
  <footer style={{ textAlign: "center", borderTop: "1px solid #ccc" }}>
    <span> &copy; Ryan Maas 2017</span>
  </footer>
)

export default Footer
