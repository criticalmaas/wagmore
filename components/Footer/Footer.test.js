import setUpTest from "../../utils/setUpTests.js"
import React from "react"
import { shallow } from "enzyme"
import Footer from "./Footer"

const footer = shallow(<Footer />)

describe("Footer", () => {
  it("renders correctly", () => {
    expect(footer).toMatchSnapshot()
  })
})
