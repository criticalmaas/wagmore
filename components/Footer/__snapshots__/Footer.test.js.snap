// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <span>
         © Ryan Maas 2017
      </span>,
      "style": Object {
        "borderTop": "1px solid #ccc",
        "textAlign": "center",
      },
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": " © Ryan Maas 2017",
      },
      "ref": null,
      "rendered": " © Ryan Maas 2017",
      "type": "span",
    },
    "type": "footer",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <span>
           © Ryan Maas 2017
        </span>,
        "style": Object {
          "borderTop": "1px solid #ccc",
          "textAlign": "center",
        },
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": " © Ryan Maas 2017",
        },
        "ref": null,
        "rendered": " © Ryan Maas 2017",
        "type": "span",
      },
      "type": "footer",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
